lib/read/setup.js

```diff
@@ -1,59 +1,78 @@
 const path = require('path');
 const fs   = require('fs');
-const os   = require('os');
-
+const yaml = require('js-yaml');
 const git  = require('simple-git');
-const Infra     = require('../infra/infrastructure');
-
 const Connector = require('infra.connectors');
 
+const Infra     = require('../infra/infrastructure');
+
 // Handles reading and processing setup block of stepfile
 class Setup {
 
-    async setup(doc, stepFile)
-    {
-        let provider = Object.keys(doc.setup)[0];
-        let cwd = provider === 'local' ? path.join(process.cwd(), path.dirname(stepFile), 'docable_results') : '.';
-
-        let docPath = path.dirname(stepFile);
-        // If using git repo, clone repo and update cwd/docPath
-        if( Object.keys(doc.setup).includes('git') )
-        {
-            console.log(`Cloning or pulling: ${doc.setup.git}`);
-            docPath = await this.cloneOrPull( doc.setup.git, path.dirname(stepFile) );
-            cwd = docPath;
+    /**
+     * Setup the env needed for execution
+     * @param {String} doc path to steps.yml to inline md file
+     * @param {String} html path to html, if doc is steps.yml
+     */
+    async setup(doc, html) {
+        let configString;
+
+        // if step
+        if (this.html) {
+            configString = await fs.promises.readFile(html);
```

Comment:
Need to make sure this is returning the right thing? Might need `{ encoding: 'utf-8' }`

> The Promise is resolved with the contents of the file. If no encoding is specified (using options.encoding), the data is returned as a Buffer object. Otherwise, the data will be a string.

Reply:
fixed in e0d0973.